pep-0698.rst

```diff
@@ -147,16 +147,16 @@ Many popular programming languages support override checks. For example:
 Runtime Override Checks in Python
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Today, there is an `Overrides library <https://pypi.org/project/overrides/>`_
-that provides decorators ``@overrides`` (sic) and ``@final`` and will enforce
+Today, there is an `Overrides library <https://pypi.org/project/overrides/>`__
```

Comment:
What do the double-underscores do?

It seems like single underscore works for inline links, as far as I can tell from (the rst quickref)[https://docutils.sourceforge.io/docs/user/rst/quickref.html] double-underscores are used for anonymous hyperlinks, which I think is not relevant here?

But I'll be honest RST confuses me so I may be wrong

Reply:
Great question. Reaching back to my first learning of reST, I had thought they were necessary for inline, direct links to places.

Now that I read the hyperlink docs more closely -- I don't think they do what I thought they did. Single underscores work fine, and I think are more correct. I'll put those back. 😬

Reply:
It makes the link anonymous, which means you can't later reference the same link target later in the document. This is useful if you have a link like `` `here <https://example.com>`__ ``. Conversely, it is not what you want if you do want to later reference the same link target, e.g.

```rst
Today, there is an `Overrides library <https://example.com>`_ ...

The `Overrides library` is a reference implementation ...
```

This is exactly the situation here, and it was a silly mistake for me to use it here, since in my other suggestions I deliberately reference the link target multiple other places.